config/preload.php

```diff
@@ -0,0 +1,5 @@
+<?php
+
+if (file_exists(dirname(__DIR__).'/var/cache/prod/App_KernelProdContainer.preload.php')) {
```

Comment:
I have read a lot about problems with the preload file under 8.0...

Reply:
well, doesn't matter if we do not create it

Reply:
And if <8.0 is not allowed 😉